docs/docs/eslint.md

```diff
@@ -2,7 +2,82 @@
 title: ESLint
 ---
 
-This is a stub. Help our community expand it.
+## Why ESLint
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+Gatsby ships with Prettier, which is a simple, opinionated code *formatter*. [ESLint](https://eslint.org) can be both a *linter* and *formatter*, meaning you can use it to check for syntactical errors as well as formatting. Prettier will work for most sites, however if you'd like to add linting capabilities *and* highly-configurable formatting you should implement ESLint into your new Gatsby project.
```

Comment:
This paragraph mentions "new" Gatsby project in the last sentence. Is there significance to that? Do you have to install ESLint at the beginning of a project?

Reply:
@shannonbux good catch! Want to just remove ~new~ here!